app/src/main/AndroidManifest.xml

```diff
@@ -36,6 +37,9 @@
             android:name=".login.views.LoginActivity"
             android:label="@string/app_name"
             android:screenOrientation="portrait" />
+        <meta-data
+                android:name="com.google.android.geo.API_KEY"
```

Comment:
Think you need to update your editor code style preferences? Android Studio => Settings => Editor => Code Style => Scheme: Project ?

Reply:
Just needed to reformat 👍